modules/account-lib/test/unit/coin/stx/transactionBuilder/transferBuilder.ts

```diff
@@ -202,6 +202,67 @@ describe('Stx Transfer Builder', () => {
       should.deepEqual(txJson.fee.toString(), '180');
     });
 
+    it('a transfer transaction signed multiple times with mid key no signer', async () => {
+      const builder = initTxBuilder();
+      builder.memo('test');
+      builder.sign({ key: testData.prv1 });
+     
+      builder.fromPubKey([testData.pub1, testData.pub2, testData.pub3]);
+      builder.numberSignatures(2);
+      const tx = await builder.build();
+      const txBuilder2 = factory.getTransferBuilder();
+      txBuilder2.from(tx.toBroadcastFormat());
+      txBuilder2.sign({ key: testData.prv3 });
+      //txBuilder2.fromPubKey([testData.pub1, testData.pub2, testData.pub3]);
```

Comment:
why is this commented?

Reply:
It is setting in  first txBuilder. It isn't required here.